Source/KakapoServer.swift

```diff
@@ -91,11 +112,15 @@ public final class KakapoServer: NSURLProtocol {
     
     /// Start loading the matched requested, the route handler will be called and the returned object will be serialized.
     override public func startLoading() {
-        KakapoServer.routers.filter { $0.canInitWithRequest(request) }.first!.startLoading(self)
+        if requestCancelled == false {
```

Comment:
no `guard`, but using `if` statement for quick entry check (60417ca9bbfda16e5fb9e84a37e0ebabc3137944)